model/src/main/java/cz/muni/fi/pa165/team/match/containers/MatchListContainer.java

```diff
@@ -0,0 +1,61 @@
+package cz.muni.fi.pa165.team.match.containers;
+
+import cz.muni.fi.pa165.team.match.TeamMatch;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * @author Tomas Smid <smid.thomas@gmail.com>
+ */
+public class MatchListContainer
+{
+    private Collection<TeamMatch> matches;
+
+    private Map<TeamMatch, Long> homeGoals;
```

Comment:
tohle cely neni moc dobra struktura, ty potřebuje přidat informaci k zápasu o tom, kolik dali gólů, ale to tady vůbec neděláš...